### DISCUSSION TITLE
About homebrew-cask CI

### DISCUSSION THREAD

**Discussion by mohzulfikar** [Homebrew/discussions#3574] [Casks] [answered]
Hi there, I want to ask about how the CI workflow on the cask project works. Are the workflows on these four repositories:
1. [homebrew-cask](https://github.com/Homebrew/homebrew-cask/blob/master/.github/workflows/ci.yml)
2. [homebrew-cask-drivers](https://github.com/Homebrew/homebrew-cask-drivers/blob/master/.github/workflows/ci.yml)
3. [homebrew-cask-fonts](https://github.com/Homebrew/homebrew-cask-fonts/blob/master/.github/workflows/ci.yml)
4. [homebrew-cask-version](https://github.com/Homebrew/homebrew-cask-versions/blob/master/.github/workflows/ci.yml)

Synchronized automatically? If so, then when I want to modify a piece of code in the cask workflow, on what repository should I make the changes? Thank you.

**Comment by carlocab** [comment, marked as answer]
The contents of `.github` should be synced from `homebrew-cask`, IIRC.

**Reply by mohzulfikar** [reply]
I see, so the repository that I should modify is homebrew-cask right? Thank you for the confirmation.

**Reply by Rylan12** [reply]
To clarify, it's not the entire contents of `.github` that are synced. It's only the files listed in [here](https://github.com/Homebrew/homebrew-cask/blob/536ecfd6076d933c82fe16c0b600e666f6fdd31c/.github/actions/sync/templates.rb#L17-L25).

> I see, so the repository that I should modify is homebrew-cask right?

Yes, once changes to the files listed in the link above are merge, new PRs will be automatically opened by @BrewTestBot in the rest of the cask repos.
